package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redux-recompose",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

Comment:
This version must be upgrade only before publish. Why not 2.0.1? @mvbattan what u think?